RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -108,6 +112,19 @@ namespace pixelgpudetails {
   }
 
   SiPixelRawToClusterGPUKernel::~SiPixelRawToClusterGPUKernel() {
+    // free the host memory
```

Comment:
I guess we were just leaking these before ?

Reply:
Yes, I suppose we were...